src/explore-education-statistics-admin/src/pages/release/content/components/__tests__/EditableKeyStatText.test.tsx

```diff
@@ -3,15 +3,16 @@ import { KeyStatisticText } from '@common/services/publicationService';
 import { render, screen, waitFor } from '@testing-library/react';
 import { noop } from 'lodash';
```

Comment:
not part of your change but we should optimize this import while you're here (we're currently importing the entire lodash lib here which is big) 

```
import noop from 'lodash/noop';
```

Reply:
Made the same change in `EditableKeyStatDataBlock.test.tsx` and `CancelAmendmentModal.test.tsx`